media-gfx/noteshrink/noteshrink-0.1.1.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="Convert scans of handwritten notes to beautiful, compact PDFs"
+DESCRIPTION="Noteshrink is an utility that is used to convert scans of notes \
+to compact PDFs.
+"
+HOMEPAGE="https://github.com/mzucker/noteshrink"
+COPYRIGHT="2016 Matt Zucker"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/mzucker/noteshrink/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="2a29c09768973e688b693b61337c6c49384e8123cf88824dba335cc8c4ed2ca8"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Comment:
Should be "any"

Reply:
Are you sure? Some of the required are untested or can't be build on other architectures (e.g. scipy).

Reply:
But the recipe itself doesn't contain any compiled code, so should be any. If a dependency cannot be built in the architecture it's already reflected and the package won't be installed because the dependency doesn't work. But the package itself could still work if someone makes a version of the dependency that works

Reply:
I would at least add a comment somewhere that it doesn't work on gcc2hybrid due to missing dependencies, maybe create an issue for those not available (if not done already)
Or even try to fix the missing dependencies :)

Reply:
I added comment to the recipe. It seems on gcc2hybrid we can't install numpy_python, pillow_python and scipy_python which are needed for the program to work properly.

Reply:
For pillow and numpy I have already a local recipe, but not for scipy (yet), still fighting with python over this :), but yes, for now a comment for that would be good